src/main/resources/application.yml

```diff
@@ -47,5 +59,5 @@ soapWebservice:
   user: Admin
   password: pa$$w0rd
 
-allow:
-  origin: *
+#allow:
```

Comment:
We have to be careful with removing this header, because this might introduce CORS problems again. I am going to change this value to a wildcard.

Reply:
Okay, sure